pom.xml

```diff
@@ -80,6 +100,11 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>2.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>commons-beanutils</groupId>
+            <artifactId>commons-beanutils</artifactId>
+            <version>1.9.4</version>
```

Comment:
It is better to use properties for each version tag, like you did with jjwt-jackson, io.jsonwebtoken etc